test/mock/kernel/idt_mock.zig

```diff
@@ -39,7 +39,7 @@ const NUMBER_OF_ENTRIES: u16 = 256;
 const TABLE_SIZE: u16 = @sizeOf(IdtEntry) * NUMBER_OF_ENTRIES - 1;
 
 pub fn openInterruptGate(index: u8, handler: InterruptHandler) IdtError!void {
-    return mock_framework.performAction("openInterruptGate", IdtError!void, index, handler);
+    return mock_framework.performAction("openInterruptGate", IdtError!void, .{ index, handler });
```

Comment:
Space at the end of the .{}? Is this what zig fmt outputs

Reply:
It is

Reply:
When I run zig fmt, it removes the spaces.

Reply:
Hmm it doesn't when I try. What command are you running?

Reply:
Found that `zig fmt` is inconsistent with formatting the `.{}`. So will fix later